magda-web-server/src/index.ts

```diff
@@ -146,6 +146,7 @@ app.get("/server-config.js", function(req, res) {
         disableAuthenticationFeatures: argv.disableAuthenticationFeatures,
         baseUrl: addTrailingSlash(argv.baseUrl),
         apiBaseUrl: apiBaseUrl,
+        baseExternalUrl: argv.baseExternalUrl,
```

Comment:
Do we need `addTrailingSlash` it like `baseExternalUrl: addTrailingSlash(argv.baseExternalUrl)`?

Reply:
Well, it works like it is, but I suppose it'll work with a trailing slash too - I'll add one and see.